src/EdFi.AnalyticsMiddleTier.Tests/Classes/ClassPeriodDim.cs

```diff
@@ -11,12 +11,12 @@ namespace EdFi.AnalyticsMiddleTier.Tests.Classes
     public class ClassPeriodDim
     {
         public string ClassPeriodKey { get; set; } //(string, not null)
+        public string SectionKey { get; set; }
```

Comment:
The test associated to this field wasn't working. In order to the tests to work, the field must be defined, otherwise the test is not doing anything.